libraries/AC_Planck/AC_Planck.cpp

```diff
@@ -135,10 +135,12 @@ void AC_Planck::request_takeoff(const float alt)
     0,0,0,0,0);
 }
 
-void AC_Planck::request_alt_change(const float alt)
+void AC_Planck::request_alt_change(const float alt, const float rate_up, const float rate_down)
 {
   //Only altitude is valid
   uint8_t valid = 0b00000100;
+  uint32_t max_rates = uint8_t(rate_up) << 8;
```

Comment:
Its not clear whats happening here. Is `rate_up` in cms, and this is truncating the float?  I would be more explicit, like change to `rate_up_cms`.  You could also do this `max_rates` assignment in one line.